build.gradle

```diff
@@ -171,6 +174,14 @@ task fatJar(type: Jar) {
     with jar
 }
 
+tasks.named('shadowJar', ShadowJar) {
+    enableRelocation true
+    relocationPrefix "io.dgraph.dgraph4j.shaded"
+    relocate 'google', 'io.dgraph.dgraph4j.shaded.google'
```

Comment:
The `ShadowJar` relocates all dependencies, but for some reason it does not automatically relocate the `google` package. Only with this line it also includes that package.

Classes in the `io.dgraph.dgraph4j` are not relocated, only dependencies.

I'd recommend to use the classifier (e.g. `shaded`):

    implementation "io.dgraph:dgraph4j-all:24.0.0:shaded"

rather than giving it a new archive name / artifact name:

    implementation "io.dgraph:dgraph4j-shaded:24.0.0"

as this better describes of what jar this is the shaded variant.